pkg/command/service/measure.go

```diff
@@ -626,21 +631,25 @@ kperf service measure --svc-perfix svc --range 1,200 --namespace ns --concurrenc
 				fmt.Printf("Percentile99: %fs\n", measureFinalResult.Result.P99)
 
 				current := time.Now()
-				rawPath := fmt.Sprintf("/tmp/%s_%s", current.Format("20060102150405"), "raw_ksvc_creation_time.csv")
+				outputLocation, err := utils.CheckOutputLocation(measureArgs.output)
+				if err != nil {
+					fmt.Printf("failed to check measure output location: %s\n", err)
```

Comment:
Sorry, another comment. Maybe make a "return" here? if output is illegal, it's not necessary to do below.
`return fmt.Errorf("failed to check measure output location: %s\n", err)`

Reply:
If we don't return and just issue the error message, then the end-user can get more info about how many output files will be generated (even there's an error). I think this information will be helpful sometimes. What do you think?

Reply:
@ShiningTian   That's OK,  LGTM. @zhanggbj Any comments on this?